cli/local-net

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+
+import subprocess
+import threading
+import sys
+import getopt
+import os
+from random import *
+import time
+import json
+
+colors = {
+    "red": 91,
+    "green": 92,
+    "yellow": 93,
+    "lightPurple": 94,
+    "purple": 95,
+    "cyan": 96,
+    "lightGray": 97,
+}
+worker_colors = {}
+
+
+def config(network, boot_nodes, rpc_nodes):
+    if not os.path.isdir('./local-net'):
+        os.mkdir('./local-net')
```

Comment:
This kind of conflicts with line 28
The script name is `local-net` and also the folder for the local data is called `local-net`   so if you run inside the `cli` folder, it will fail. 
Your "docs" in this pr tells to run `./local-net binary` but at the #777 it shows `cli/local-net`  
In any case, please add some doc to README explaining how to run it so the paths don't conflict.

Reply:
Yes. You are right. I will add some docs but you can run this anywhere but in the cli directory. I don't know if I want to change the script name just for that.

Reply:
Just having some docs should be fine.
Or change the folder to `local-net_data` or something would also "fix".

Reply:
Added docs and changed directory to `local-net-data`